docs/src/main/asciidoc/_elytron/migrate/Simple_SSL_Migration.adoc

```diff
@@ -30,7 +30,7 @@ generated using keytool is JKS:
 +
```

Comment:
@theashiot In the sentence above this line, we should s/Default type of keystore generated using keytool is JKS/Default type of keystore generated using keytool is PKCS12

Reply:
done